.github/workflows/ci-build-image.yml

```diff
@@ -80,6 +84,7 @@ jobs:
   integration-tests:
     runs-on: ubuntu-latest
     needs: [build-image]
+    if: ${{ inputs.skip-itests && inputs.skip-itests != 'false' }}
```

Comment:
Is this syntax correct for boolean parameters? the quotes make me think it's a string

Reply:
It wasn't clear to me if the system will actually provide a boolean at runtime or if it will be a bool-ish string:

https://github.com/actions/runner/issues/1483

I can make it `${{ ! inputs.skip-itests }}` (ie treat it as a boolean) and see if it works?